x-pack/packages/kbn-cloud-security-posture/common/utils/helpers.ts

```diff
@@ -62,3 +62,43 @@ export const buildEntityFlyoutPreviewQuery = (field: string, queryValue?: string
     },
   };
 };
+
+export const buildEntityAlertsQuery = (field: string, queryValue?: string, size?: number) => {
+  return {
+    size: size || 0,
+    _source: false,
+    fields: [
+      'kibana.alert.rule.uuid',
+      'signal.rule.name',
+      'signal.rule.severity',
+      'kibana.alert.reason',
+    ],
+    query: {
+      bool: {
+        filter: [
+          {
+            bool: {
+              must: [],
+              filter: [
+                {
+                  match_phrase: {
+                    [field]: {
+                      query: queryValue,
+                    },
+                  },
+                },
+              ],
+              should: [],
+              must_not: [],
+            },
+          },
+          {
+            terms: {
+              'kibana.alert.workflow_status': ['open', 'acknowledged'],
```

Comment:
I believe there should be exported constants of existing statuses, we should use these instead of hardcoding their values. same for the field where these statuses are stored. Ideally we shouldn't have this as hardcoded strings and shouldn't create our own constants